ansible/inventory/teleop/group_vars/control_machine.yml

```diff
@@ -24,6 +24,10 @@ arm_ip_left: "" # Specify the IP address of left arm
 arm_ip_right: "" # Specify the IP address of right arm
 ur_robot_type: "ur10e" # Specify ur robot type: ur10, ur10e, ur5, ur5e
 biotacs: true # Whether the hands will have biotac fingertips
+bag_all_data_from: ["phantomhand_A", "phantomhand_B"]
+remote_bag_user: "shadowop"
+remote_bag_storage_IP: "10.5.10.59"
```

Comment:
seems dangerous to hard code an ip like this? Can we parametrize it?

Reply:
This IP address was allocated in London office router to the NAS. Idea was to have usable defaults to make new hand setup easier.